src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,42 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+  private final List<Car> cars;
+
+  public Cars(Names names) {
+    cars = new ArrayList<>();
+    names.getNames()
+        .forEach(name -> cars.add(new Car(name)));
+  }
+
+  public int size() {
+    return cars.size();
+  }
+
+  public Car at(int index) {
+    return cars.get(index);
+  }
+
+  public void moveAll(RaceEvaluator raceEvaluator) {
+    cars.forEach(car -> car.tryMove(raceEvaluator));
```

Comment:
```suggestion
    cars.forEach(car -> {
      if (raceEvaluator.evaluate()) {
        car.addScore(1);
      }
    });
```

복잡한 `raceEvaluator`에 대한 의존성을 깊은 뎁스까지 전달하는게 확장에 유리할까요?
`Car`객체가 가장 단순하게 받고 싶은 메시지는 무엇일까요?